src/components/ProseCode.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="my-8 rounded-lg"
+    class="mb-8 rounded-lg"
```

Comment:
This was messing up the layout so removed the margin from the top. I have checked that this doesn't affect existing published documentation.